File: Python/beginner/1589.py
# https://judge.beecrowd.com/pt/problems/view/1589
# Bob Conduite

# Main Code
cases = int(input())

for i in range(cases):
    try:
        wire1, wire2 = map(int, input().split())
        print(wire1 + wire2)
    except EOFError:
        break
    except ValueError:
        break
        
# By: Fernando Serra
# https://github.com/fernandosserra

File: Haskell/beginner/1589.hs
-- https://judge.beecrowd.com/pt/problems/view/1589
-- Bob Conduite

-- Main function
main :: IO ()
main = do
    n <- readLn :: IO Int -- Reads the number of lines or cases
    mapM_ (\_ -> do
        line <- getLine
        let [r1, r2] = map read (words line) :: [Int] -- Get the wires diameters (or radii)
        print (r1 + r2) -- Returns the conductor size
        ) [1..n] -- Iterates between 1 and n

-- By Fernando Serra
-- https://github.com/fernandosserra